client/src/components/Nav/index.js

```diff
@@ -43,6 +43,10 @@ class Navbar extends Component {
               <Link to="#!">
                 <img className="brand-logo" id="logo" alt="logo" src={logo} />
               </Link>
+              {/* <div className="container">
```

Comment:
You could potentially wrap this in a condition. If this.state.email is present, show this whole block...

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator